.github/workflows/desktop-build.yml

```diff
@@ -333,20 +333,15 @@ jobs:
         with:
           submodules: recursive
 
-      - name: Restore Qt ${{matrix.qt_version}} for ${{matrix.target}} from cache
-        id: cache-qt
-        uses: actions/cache@v1 # Intentionally v1, based on jurplel documentation
-        with:
-          key: QtCache-${{matrix.qt_version}}-${{matrix.target}}
-          path: '${{github.workspace}}/../Qt'
-
       - name: Install Qt ${{matrix.qt_version}} for ${{matrix.target}}
         uses: jurplel/install-qt-action@v3
         with:
-          cached: ${{steps.cache-qt.outputs.cache-hit}}
+          cache: true
           version: ${{matrix.qt_version}}
           arch: win${{matrix.arch}}_${{matrix.qt_arch}}
+          setup-python: false
           modules: ${{matrix.qt_modules}}
+          tools: tools_openssl_${{matrix.arch == 64 && 'x64' || 'x86'}}  # ternary operator styled workaround to get correct arch for openssl
```

Comment:
this isn't as much a workaround, it'd be simpler to just place this value in the matrix or simply use the vcpkg_default_triplet which holds exactly the same value

Reply:
You don't like short-circuit evaluation here?

I reworked the values a bit to use more general terms and tools configuration is now done in the matrix section, too.
Is that better?